noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -111,6 +103,7 @@ contract Token {
         // docs:end:initialize_decimals
     }
     // docs:end:constructor
+
```

Comment:
It was bothering me that there were no spaces between functions so I sneaked it in.